### PR TITLE
Fixes #1777: Documentation uses deprecated functions - https://neo4j.com/labs/apoc/4.2/virtual/virtual-nodes-rels/

### LINKED ISSUE #1777: Documentation uses deprecated functions - https://neo4j.com/labs/apoc/4.2/virtual/virtual-nodes-rels/
## Expected Behavior (Mandatory)

NOTE: Documentation buttons lead to github as the place to report documentation issues, but this might not be the best path?

Example cypher code in documentation works as provided as functionality implies with "Run in Neo4j Browser" and "Copy to Clipboard"

[documentation page](https://neo4j.com/labs/apoc/4.2/virtual/virtual-nodes-rels/ )
references the (deprecated I assume?) non existent function 

`apoc.coll.groupBy`

## Actual Behavior (Mandatory)

```
Neo.ClientError.Statement.SyntaxError
Unknown function 'apoc.coll.groupBy' (line 3, column 6 (offset: 156))
"WITH apoc.coll.groupBy(countryNodes, "name") AS countries"
    ^
```

## How to Reproduce the Problem

Use cypher from the 4.x [documentation page](https://neo4j.com/labs/apoc/4.2/virtual/virtual-nodes-rels/ )

### Simple Dataset (where it's possibile)

```
MATCH (p:Person) WITH collect(distinct p.country) as countries
WITH [cName IN countries | apoc.create.vNode(['Country'],{name:cName})] as countryNodes
WITH apoc.coll.groupBy(countryNodes,'name') as countries
MATCH (p1:Person)-[:KNOWS]->(p2:Person)
WITH p1.country as cFrom, p2.country as cTo, count(*) as count, countries
RETURN countries[cFrom] as from, countries[cTo] as to, apoc.create.vRelationship(countries[cFrom],'KNOWS',{count:count},countries[cTo]) as rel;
```

### Steps (Mandatory)

  1. Use cypher from documentation

## Screenshots (where it's possibile-[:CORRECT_SPELLING]->(possible) )

deprecated function

## Specifications (Mandatory)

4.x documentation page

### Versions

4.x documentation page

### PR DESCRIPTION
**Pull request by vga91**
Fixes #1777

- Improved `virtual-nodes-rels.adoc` and fixed wrong function

### CODE REVIEW

File: docs/asciidoc/modules/ROOT/pages/virtual/virtual-nodes-rels.adoc [hunk 1]
```diff
@@ -53,27 +53,50 @@ include::example$generated-documentation/apoc.create.virtualPath.adoc[]
 
 .Virtual node and virtual relationship `vNode`, `vRelationship`
 
+From this simple dataset:
```
Comment (JMHReif) [inline docs/asciidoc/modules/ROOT/pages/virtual/virtual-nodes-rels.adoc:56]:
```suggestion
From this simple data set:
```

File: docs/asciidoc/modules/ROOT/pages/virtual/virtual-nodes-rels.adoc [hunk 2]
```diff
@@ -53,27 +53,50 @@ include::example$generated-documentation/apoc.create.virtualPath.adoc[]
...
+
+[source,cypher]
+----
+CREATE(a:Person)-[r:ACTED_IN]->(b:Movie)
+----
+
+we can do:
```
Comment (JMHReif) [inline docs/asciidoc/modules/ROOT/pages/virtual/virtual-nodes-rels.adoc:63]: How does the above example code tie to the below example code? Is there something I'm missing?

Comment (vga91) [reply docs/asciidoc/modules/ROOT/pages/virtual/virtual-nodes-rels.adoc:66]: I'm sorry, very bad copy paste mistake, I forget to change it :(

File: docs/asciidoc/modules/ROOT/pages/virtual/virtual-nodes-rels.adoc [hunk 3]
```diff
@@ -53,25 +53,50 @@ include::example$generated-documentation/apoc.create.virtualPath.adoc[]
...
 MATCH (p1:Person)-[:KNOWS]->(p2:Person)
 WITH p1.country as cFrom, p2.country as cTo, count(*) as count, countries
 RETURN countries[cFrom] as from, countries[cTo] as to, apoc.create.vRelationship(countries[cFrom],'KNOWS',{count:count},countries[cTo]) as rel;
 ----
 
+image::apoc.create.vRelationshipTwo.png[scaledwidth="100%"]
+
 That's of course easier with `apoc.nodes.group`.
 
 From a simple dataset
```
Comment (fbiville) [inline docs/asciidoc/modules/ROOT/pages/virtual/virtual-nodes-rels.adoc:102]:
```suggestion
From a simple data set
```

**Review by fbiville — APPROVED** [review]

**Pull request merged**